### ISSUE TITLE
Is there a way to specify the port?

### ISSUE THREAD

**Issue by ramswaroop** [caelum/mamute#272]
Is there any way to make mamute run on a specific port?

**Comment by felipeweb** [comment]
Hi Ram you can specify the PORT environment variable on your machine :)

Em qua, 17 de ago de 2016 às 12:36, Ram swaroop <notifications@github.com>
escreveu:

**Comment by ramswaroop** [comment]
Hi,

You mean do I have to setup a proxy server like Apache or nginx with port 80?

**Comment by felipeweb** [comment]
No, if you use linux or mac open your terminal and type ```export
PORT=80``` or any por that you want

Em qua, 17 de ago de 2016 às 13:20, Ram swaroop <notifications@github.com>
escreveu:
